packages/databricks-vscode/src/telemetry/index.ts

```diff
@@ -0,0 +1,144 @@
+import TelemetryReporter from "@vscode/extension-telemetry";
+import {DatabricksWorkspace} from "../configuration/DatabricksWorkspace";
+import {isDevExtension} from "../utils/developmentUtils";
+import {
+    DEV_APP_INSIGHTS_CONFIGURATION_STRING,
+    EventType,
+    EventTypes,
+    PROD_APP_INSIGHTS_CONFIGURATION_STRING,
+} from "./constants";
+import crypto from "crypto";
+import bcrypt from "bcrypt";
+
+export {Events, EventTypes} from "./constants";
+
+let telemetryReporter: TelemetryReporter | undefined;
```

Comment:
Not a fan of globals. Why can't we just instantiate a TelemetryReporter in `extension.ts`?

Reply:
I'd prefer to sub class `TelemetryReporter` and create a single instance instead.